src/errorpage.cc

```diff
@@ -1523,3 +1523,12 @@ ErrorPage::ValidateStaticError(const int page_id, const SBuf &inputLocation)
     anErr.validate();
 }
 
+std::ostream &
+operator <<(std::ostream &os, ErrorState *err)
```

Comment:
```suggestion
operator <<(std::ostream &os, const ErrorState *err)
```

Technically, there is one more `const` missing here (for the `err` pointer itself), but Squid code does not usually use it, I myself have not gotten into the habit of adding it, so I am not suggesting it. Your call though.